examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -3,87 +3,86 @@
+
+- the **exposure** is the duration of the insurance coverage of a given policy,
+  in years.
+
+- the claim **frequency** is the number of claims divided by the exposure,
+  typically measured in number of claims per year.
+
+In this dataset, each sample corresponds to an insurance policy. Available
+features include driver age, vehicle age, vehicle power, etc.
+
+Our goal is to predict the expected frequency of claims following car accidents
+for a new policyholder given the historical data over a population of
+policyholders.
 
 .. [1]  A. Noll, R. Salzmann and M.V. Wuthrich, Case Study: French Motor
-    Third-Party Liability Claims (November 8, 2018).
-    `doi:10.2139/ssrn.3164764 <http://dx.doi.org/10.2139/ssrn.3164764>`_
+    Third-Party Liability Claims (November 8, 2018). `doi:10.2139/ssrn.3164764
+    <http://dx.doi.org/10.2139/ssrn.3164764>`_
 
 """
 print(__doc__)
-
 # Authors: Christian Lorentzen <lorentzen.ch@gmail.com>
 #          Roman Yurchak <rth.yurchak@gmail.com>
+#          Olivier Grisel <olivier.grisel@ensta.org>
 # License: BSD 3 clause
 import warnings
-
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 
-from sklearn.datasets import fetch_openml
-from sklearn.dummy import DummyRegressor
-from sklearn.compose import ColumnTransformer
-from sklearn.linear_model import Ridge, PoissonRegressor
-from sklearn.model_selection import train_test_split
-from sklearn.pipeline import make_pipeline
-from sklearn.preprocessing import FunctionTransformer, OneHotEncoder
-from sklearn.preprocessing import OrdinalEncoder
-from sklearn.preprocessing import StandardScaler, KBinsDiscretizer
-from sklearn.ensemble import RandomForestRegressor
-from sklearn.utils import gen_even_slices
-from sklearn.metrics import auc
-
-from sklearn.metrics import mean_squared_error, mean_absolute_error
-from sklearn.metrics import mean_poisson_deviance
-
-
-def load_mtpl2(n_samples=100000):
-    """Fetch the French Motor Third-Party Liability Claims dataset.
 
-    Parameters
-    ----------
-    n_samples: int or None, default=100000
-      Number of samples to select (for faster run time). If None, the full
-      dataset with 678013 samples is returned.
-    """
+##############################################################################
+# The French Motor Third-Party Liability Claims dataset
+# -----------------------------------------------------
+#
+# Let's load the motor claim dataset. We ignore the severity data for this
+# study for the sake of simplicitly:
 
-    # freMTPL2freq dataset from https://www.openml.org/d/41214
-    df = fetch_openml(data_id=41214, as_frame=True)['data']
+from sklearn.datasets import fetch_openml
 
-    # unquote string fields
-    for column_name in df.columns[df.dtypes.values == np.object]:
-        df[column_name] = df[column_name].str.strip("'")
-    if n_samples is not None:
-        return df.iloc[:n_samples]
-    return df
 
+df = fetch_openml(data_id=41214, as_frame=True).frame
```

Comment:
this is raising 

```
/home/circleci/project/sklearn/datasets/_openml.py:655: UserWarning: Version 1 of dataset freMTPL2freq is inactive, meaning that issues have been found in the dataset. Try using a newer version from this URL: https://www.openml.org/data/v1/download/20649148/freMTPL2freq.arff
```

Reply:
Yes but unfortunately there is nothing we can do about it apart from contacting the original uploader. Let me do that.